playbook/app/pb_kits/playbook/pb_file_upload/docs/_file_upload_custom.html.erb

```diff
@@ -1,7 +1,5 @@
-<%= pb_rails("file_upload", props: {id: "import_file"}) do %>
-  <input
-    id="import_file"
-    type="file"
-    name="import_file"
-  />
-<% end %>
\ No newline at end of file
+<%= pb_form_with(method: :get, multipart: true) do |form| %>
+  <%= form.text_field :name, style: "width: 140px", props: { placeholder: "Name" } %>
```

Comment:
As discussed in 3/28 deep dive dailies review - name field is present as it is required for the playground demo to work and shows how this kit functions within a form/form group. But I can definitely remove that field from this faux-functional demo if simpler is better for a kit example (or just better aesthetically). Would look like:
<img width="531" alt="Remove name field from example area" src="https://github.com/powerhome/playbook/assets/83474365/a034cace-6d70-47af-83f3-c9ade9b91020">

Reply:
Yeah I'd go ahead and remove the name field. @ElisaShapiro

Reply:
I'm fine logging the "File" that the user uploads.

Reply:
Just pushed a commit that implemented this (as discussed in Connect). Thanks for the suggestion!